Raven.Studio.Html5/App/commands/database/documents/getCollectionLabelCommand.ts

```diff
@@ -0,0 +1,35 @@
+﻿import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+
+class getCollectionLabelCommand extends commandBase {
```

Comment:
I couldn't find any examples of using the StudioConfig file. If I can figure out how its done, I'll try to make these changes.